src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer {
```

Comment:
Now, this class is not parameterized (we didn't declare what type is playing the role of <T> from MachineProducer. Let's fix that